e2e/tests/plugins/notebook/restrictedNotebook.e2e.spec.js

```diff
@@ -146,13 +49,16 @@ test.describe('Restricted Notebook', () => {
         // notbook tree object exists
         expect.soft(await restrictedNotebookTreeObject.count()).toEqual(1);
 
-        // Click text=Remove
+        // Click Remove Text
         await page.locator('text=Remove').click();
-        // Click text=OK
+
+        //Wait until Save Banner is gone
         await Promise.all([
-            page.waitForNavigation(/*{ url: 'http://localhost:8080/#/browse/mine?tc.mode=fixed&tc.startBound=1653671067340&tc.endBound=1653672867340&tc.timeSystem=utc&view=grid' }*/),
-            page.locator('text=OK').click()
+            page.waitForNavigation(),
+            page.locator('text=OK').click(),
+            page.waitForSelector('.c-message-banner__message')
```

Comment:
should we use an aria label here?

Reply:
If there's one available. This is just a general refactor to add some stability to these tests and make them work on Firefox